src/main/java/org/apache/commons/csv/ExtendedBufferedReader.java

```diff
@@ -132,10 +132,24 @@ int lookAhead() throws IOException {
      */
     char[] lookAhead(final int n) throws IOException {
         final char[] buf = new char[n];
```

Comment:
The local var is not needed anymore right?

Reply:
Actually, it's still used in `CSVFormat#printWithEscapes`.  I am happy to remove fully in another PR, but wanted to keep change as small as possible.